src/test/java/io/jenkins/plugins/uleska/toolkitscanner/UleskaToolkitScannerFactoryTest.java

```diff
@@ -0,0 +1,21 @@
+package io.jenkins.plugins.uleska.toolkitscanner;
+
+import hudson.model.TaskListener;
+import org.junit.Test;
+
+import static org.junit.Assert.assertNotNull;
+
+public class UleskaToolkitScannerFactoryTest {
+
+
+    @Test
+    public void testBuildReturnsIntanceOfScanner() throws Exception {
```

Comment:
Minor issue, spelling mistake, should be "Instance" not "Intance"

Reply:
I have change this, thanks.